### PR TITLE
PODS-10010:Penalties and charges page changes

### PR DESCRIPTION
**Pull request by dynamicwave4**

### CODE REVIEW

File: app/config/FrontendAppConfig.scala [hunk 1]
```diff
@@ -107,7 +107,11 @@ class FrontendAppConfig @Inject()(configuration: Configuration, servicesConfig:
   lazy val aftOverviewUrl: String = s"$aftUrl${configuration.get[String](path = "urls.aftOverview")}"
   lazy val isAftNonZero: String = s"$aftUrl${configuration.get[String](path = "urls.isAftNonZero")}"
   lazy val psaFinancialStatementUrl: String = s"$aftUrl${configuration.get[String](path = "urls.psaFinancialStatement")}"
-  lazy val podsNewFinancialCredits: Boolean = configuration.getOptional[Boolean]("pods.new.financial.credits").getOrElse(false)
+  lazy val podsNewFinancialCredits: Boolean = {
+
+    val bool = configuration.getOptional[Boolean]("pods.new.financial.credits").getOrElse(false)
```
Comment (jamdade-sharad) [inline app/config/FrontendAppConfig.scala:112]: we can remove **val bool** and return directly **configuration.getOptional[Boolean]("pods.new.financial.credits").getOrElse(false)**

**Review by jamdade-sharad — COMMENTED** [review]
we can remove **val bool**

File: app/config/FrontendAppConfig.scala (hunk 1 above)
Comment (dynamicwave4) [reply app/config/FrontendAppConfig.scala]: Removed

**Review by jamdade-sharad — APPROVED** [review]
lgtm

**Pull request merged**